.github/workflows/release_charts.yml

```
@@ -28,15 +30,21 @@ jobs:
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
-      - name: Add repositories
+      - name: Install YQ
+        uses: mikefarah/yq@master
+
+      - name: Get outputs
+        id: vars
         run: |
-          for dir in $(ls -d charts/*/); do
-            helm dependency list $dir 2> /dev/null | grep -v oci | tail +2 | head -n -1 | awk '{ print "helm repo add " $1 " " $3 }' | while read cmd; do $cmd; done
-          done
+          GIT_VERSION=$(git describe --tags --always)
+          echo "version=${GIT_VERSION}" >> $GITHUB_OUTPUT
 
-      - name: Helm Deps
+      - name: Setup charts
         run: |
-          for dir in $(ls -d charts/*); do
+          for dir in $(ls -d charts/*/); do
+            yq -i '.version = "${{ steps.vars.outputs.version }}"' "$dir/Chart.yaml"
+            yq -i '.appVersion = "${{ steps.vars.outputs.version }}"' "$dir/Chart.yaml"
+            helm dependency list $dir 2> /dev/null | grep -v oci | tail +2 | head -n -1 | awk '{ print "helm repo add " $1 " " $3 }' | while read cmd; do $cmd; done
```

Not only `oci://` but also local `file://` repos don't need `helm repo add` - https://helm.sh/docs/helm/helm_dependency/

We also have `yq` installed here already, so we can replace the old `grep/tail/head/aws` output format assumptions with a clean yaml query:
```suggestion
            yq '.dependencies[] | select(.repository | test("^https?://")) | "helm repo add " + .name + " " + .repository' $dir/Chart.yaml | sh
```

Test output with `dir=charts/kof-storage`:
```
helm repo add victoria-metrics-operator https://victoriametrics.github.io/helm-charts/
helm repo add victoria-logs-single https://victoriametrics.github.io/helm-charts/
helm repo add external-dns https://kubernetes-sigs.github.io/external-dns/
```